docs/markdown/android/checkbox.md

```diff
@@ -0,0 +1,135 @@
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/d3/66/fb/d366fb0a6b4bc150779c149aea472a24.jpg" noPadding alt="Primary example of Checkbox component" />
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />
+    - In a list, form or [Table](https://gestalt.pinterest.systems/web/table), to present users with multiple, related options where more than one option can be selected. Users must be able to select all, none or some of the presented options.
+    - In a Form, along with a [TextField](https://gestalt.pinterest.systems/android/textfield), or other spaces that are too small for a [Switch](https://gestalt.pinterest.systems/android/switch).
+    - When selection doesn’t take immediate effect and requires form submission.
+  </Group>
+
+  <Group>
+    <Dont title="When not to use" />
+    - Situations where users can only choose one out of multiple, related options. Use [RadioGroup](https://gestalt.pinterest.systems/web/radiogroup) instead.
+    - When a selection takes immediate effect, especially on mobile. Use [Switch](https://gestalt.pinterest.systems/android/switch) instead.
+    - When visually, it’s hard to tell that a checkbox turns something on or off. Use [Switch](https://gestalt.pinterest.systems/android/switch) instead.
+  </Group>
+  </TwoCol>
+
+## Best practices
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/32/33/bb/3233bb3a92c57dd29f2b5f2fd2a417f7.jpg" noPadding alt="Example of correct multi-select use" />
+    <Do title="Do" />
+    Use checkboxes for multi-selection of related list items
+  </Group>
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/1e/14/24/1e14245a26f12cf014028d24b00a6317.jpg" noPadding alt="Example of incorrect single-select use" />
+    <Dont title="Don't" />
+    Use checkboxes for one selection. Use [RadioGroup](https://gestalt.pinterest.systems/web/radiogroup) instead.
+  </Group>
+</TwoCol>
+
+<TwoCol>
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/b2/20/93/b2209377241895d0471bde7340ee85c2.jpg" noPadding alt="Example of correct single checkbox use" />
+    <Do title="Do" />
+    Use a single Checkbox in forms where the selection only takes effect after submitting the form
+  </Group>
+
+  <Group>
+    <ImgContainer src="https://i.pinimg.com/originals/e7/07/02/e70702e40416c5f4f89e92eca2bbb490.jpg" noPadding alt="Example of incorrect immediate effect use" />
+    <Dont title="Don't" />
+    Use a Checkbox to turn a state on and off with immediate effect. Use [Switch](https://gestalt.pinterest.systems/android/switch) instead.
+  </Group>
+</TwoCol>
+
+## Accessibility
+
+People use Apple and Android’s accessibility features, such as VoiceOver and TalkBack to personalize how they interact with their device. Supporting these personalizations ensures that everyone has a great user experience. See Apple and Android’s accessibility documentation for more information:
+<TwoCol>
```

Comment:
We don't usually list the iOS accessibility info on Android pages, is this a new practice?

Reply:
Not sure — I think Kate and Hector are rethinking how to handle mobile docs given the large amount of overlap/duplication. This is how it was in the doc so I'll leave as-is for now and we can address in the future — either removing references to the platform that doesn't belong, or further merging mobile pages.